app/src/internal/java/com/kickstarter/ui/activities/InternalToolsActivity.java

```diff
@@ -152,7 +153,14 @@ public void featureFlagsClick() {
 
   @OnClick(R.id.email_verification_button)
   public void emailVerificationInterstitialClick() {
-    final User user = this.environment().currentUser().getUser();
+    this.environment()
+            .currentUser()
+            .observable()
+            .filter(ObjectUtils::isNotNull)
+            .subscribe(this::startInterstitialFragment);
+  }
+
+  private void startInterstitialFragment(@NonNull final User user) {
```

Comment:
nit: you can use the method on the LoginHelper https://github.com/kickstarter/android-oss/blob/8d01e7811e4bdb18b0f8780c0fe4fc6886073ea5/app/src/main/java/com/kickstarter/libs/utils/LoginHelper.kt#L50

that method was created to not have to re-write the logic every time we need to present the interstitial, here an example of use in Java https://github.com/kickstarter/android-oss/blob/8d01e7811e4bdb18b0f8780c0fe4fc6886073ea5/app/src/main/java/com/kickstarter/ui/activities/SignupActivity.java#L82

and here example of use in kotlin https://github.com/kickstarter/android-oss/blob/8d01e7811e4bdb18b0f8780c0fe4fc6886073ea5/app/src/main/java/com/kickstarter/ui/activities/LoginActivity.kt#L104

No need to change it, the goal of having the interstitial presented on the internal tools was to be able to work both of us in parallel, and made UAT for desing easier , this comment is just an FYI :)